### DISCUSSION TITLE
Lack of Korean in the version 3.9 Beta 2

### DISCUSSION THREAD

**Discussion by ghost** [jamulussoftware/jamulus#2740] [Bug Investigation]
I've got messaged from users from Korea. There is no Korean(which translated) in this release. Can you check this out please?

**Comment by hoffie** [comment]
Sorry, it seems like we forgot adding the new resource file in yet another place. I've submitted a PR which should fix that. Maybe you can try the build artifacts from there: #2741.

Thanks for spotting this!

**Reply by ghost** [reply]
Let me check this with other users.
Thank you :)
